packages/peregrine/lib/talons/CartPage/PriceAdjustments/GiftOptions/useGiftOptions.js

```diff
@@ -23,8 +25,8 @@ import { useCartContext } from '@magento/peregrine/lib/context/cart';
  * import { useGiftOptions } from '@magento/peregrine/lib/talons/CartPage/GiftOptions/useGiftOptions';
  */
 const useGiftOptions = props => {
```

Comment:
Define default value like `(props = {})` here in case we don't receive in props.

And everywhere where you call the talon, remove the empty object from the calls.